packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -307,22 +308,22 @@ const styles = {
         margin-left: -${space[5]}px;
         margin-right: -${space[5]}px;
     `,
-    desktopVisualContainer: css`
+    desktopVisualContainer: (isNewYearBanner?: boolean) => css`
```

Comment:
You can revert this change and add a new css variable below this one

```suggestion
    desktopVisualContainerNY: css`
        ${from.tablet} {
            display: block;
            width: 500px;
            margin-left: ${space[3]}px;
        }
        ${from.desktop} {
            width: 520px;
            margin-left: ${space[5]}px;
        }
        ${from.leftCol} {
            width: 540px;
            margin-left: ${space[9]}px;
        }
    `,
```

Reply:
Yep, understood remove parameter to use 2 simpler css parts. All done.